### ISSUE TITLE
Funcionalidade [Vaga de empregos]

### ISSUE THREAD

**Issue by andrebian** [php-pr/layout-site#10]
Referência: https://github.com/php-pr/layout-site/blob/master/funcionalidades/propostas/Vagas-de-emprego.md

Sugiro o cadastro de vagas de emprego por um simples envio de email. Recrutadores não vão querer ter um login nem ter que ficar pedindo pra alguém pra publicar. Eu acredito que seria bom disponibilizar um email em que os recrutadores enviam a descrição e automaticamente o anúncio é enviado para moderação e assim que moderado aprovado/reprovado.

Penso em algo assim: jobs@phppr.net.

O que acham?

**Comment by andrebian** [comment]
Pessoal, votem se acham que o novo site deve ter uma seção de vagas de empregos.

@jeanmalves
@joridos
@nandomoreirame
@felipecwb
@kleberksms
@fatorx
@paulomartinhago

**Comment by joridos** [comment]
:+1:  Acho que sim, é legal isso :smile:

**Comment by andrebian** [comment]
:-1: 

Motivo: Acho que já tem pulverização demais sobre empregos na área de programação, só no facebook tem trocentos grupos de vagas sem contar os recrutadores que ficam pulverizando a mesma vaga em todos os grupos de desenvolvimento. Acho que se o site tivesse, seria um trabalho a mais, teríamos de definir uma espécie de filtro pra não aparecerem vagas ridículas e teríamos de ficar todo tempo validando se as vagas ainda estão abertas pois os recrutadores só querem saber de espalhar.

Por outro lado seria interessante até mesmo para a gente ter estatísticas sobre a empregabilidade do profissional PHP do PR. Só que teria de ser bem discutida...

**Comment by joridos** [comment]
Faz sentido @andrebian, o lance da analise é interesante, oque os devs + procuram é valor hora, CTL, PJ, e informações do tipo, acho interessante isso.

**Comment by andrebian** [comment]
Sim, existem os dois lados. O ruim é que seria somente mais um canal. O bom é ao longo de um tempo podermos traçar o perfil dos profissionais bem como das oportunidades. Isto seria relevante para apresentar em palestras em todo o estado, por todos os envolvidos.

**Comment by joridos** [comment]
Legal, boa.

**Comment by nandomoreirame** [comment]
:-1: 
Eu sou da opinião de que é mais um canal apenas, talvez não tem muito sentido.
E se for pra fazer podemos fazer essa implementação em outra versão, vamos fazer um MVP pra lançamento e ir implementando aos poucos com a contribuição de todos :)

**Comment by jeanmalves** [comment]
Eu acho válido pois irá movimentar bastante o site, criará mais interação.

Em 6 de outubro de 2015 14:33, Fernando Moreira <notifications@github.com>
escreveu:
